src/impl.cpp

```diff
@@ -0,0 +1,6 @@
+#include "impl.h"
+
+bool Quadro (int a, int b, int c, double& x1, double& x2)
+{
+    return ((x1 = (-b+sqrt(b*b-4*a*c))/2/a) && (x2 = (-b-sqrt(b*b-4*a*c))/2/a));
```

Comment:
where is the include of cmath?
please use std::sqrt (https://en.cppreference.com/w/cpp/numeric/math/sqrt)